server/src/shared/helpers/error.ts

```diff
@@ -2,14 +2,17 @@ import { default as uuid } from 'uuid';
 import { pathOr } from 'ramda';
 import { ValidationErrorItem } from 'joi';
 
+import { Config } from '@config/config.types';
 import { default as config } from '@config';
 
 import { CustomError as ICustomError, CustomErrorDetail as ICustomErrorDetail } from '../shared.types';
 import { ValidationError } from './validation/error';
 
+const cfg: Config = config();
```

Comment:
Misschien config hier ook beter als property op uw class zetten? Je hebt dat aangepast in app.ts